classes/class-wc-connect-taxjar-integration.php

```diff
@@ -1008,13 +1008,20 @@ public function calculate_tax( $options = array() ) {
 			$body['line_items'] = $line_items;
 		}
 
+		$skip_taxjar_request_taxes = apply_filters( 'woocommerce_services_skip_taxjar_request', false, $taxes, $body );
+
+		if ( false !== $skip_taxjar_request_taxes ) {
+			// 4th parameter set to true indicate the this filter is skipped the TaxJar request.
+			return apply_filters( 'woocommerce_services_taxjar_calculate_tax_result', $skip_taxjar_request_taxes, null, $body, true );
```

Comment:
Suggest to rename this filter to `woocommerce_services_custom_tax_result` pass `$taxes` as first parammeter to convey what shape returned value should be. Think using same filter name in different places will be more confusing. It will require developer to exemine values passed by filter to know in which place same filter was triggered.
```suggestion
return apply_filters( 'woocommerce_services_custom_tax_result', $taxes, $body );
```